packages/rspack-test-tools/tests/__snapshots__/StatsAPI.test.js.snap

```diff
@@ -52,7 +52,7 @@ Object {
         "main.js",
       ],
       "filteredModules": undefined,
-      "hash": "6264df4ebe23175182cf",
+      "hash": "333e59d94047c9b4ec6c",
```

Comment:
ChunkHash changes because browserslist-rs and browserslist have different output for `fully supports es6`, but they should be the same in the future when they updates to the same db version